etc/ci/performance/test_perf.sh

```diff
@@ -12,8 +12,14 @@ TP5N_SOURCE="https://people.mozilla.org/~jmaher/taloszips/zips/tp5n.zip"
 TP5N_PATH="page_load_test/tp5n.zip"
 if [[ ! -f "${TP5N_PATH}" ]]; then
     wget "${TP5N_SOURCE}" -O "${TP5N_PATH}"
+else
+    echo "Found existing test cases, skipping download."
+fi
+if [[ ! -f "$(dirname "${TP5N_PATH}")/tp5n/tp5n.manifest" ]]; then
+    unzip -o "${TP5N_PATH}" -d "$(dirname "${TP5N_PATH}")"
```

Comment:
And possibly some print statements around the `unzip` call so the user knows extraction is happening, e.g. `Extracting...done`.